gemma-teixido/challenge/to-do-list-fullstack/frontend/src/redux/constants/tasksList.js

```diff
@@ -0,0 +1,8 @@
+const TASKS = [
+  { id: 1, name: 'Listen new Album of Ariana Grande', completed: false },
+  { id: 2, name: 'Record Skylab\'s new hit', completed: false },
+  { id: 3, name: 'Finish redux challenge', completed: false },
+  { id: 4, name: 'Buy candy for my co-workers of Skylab', completed: false }
+];
+
+export default TASKS;
```

Comment:
You can remove this file if you connect to DDBB and don't use them.

Reply:
done!!!